src/platforms/platform.dom.js

```diff
@@ -370,6 +358,10 @@ module.exports = {
 		// https://github.com/chartjs/Chart.js/issues/2807
 		var context = item && item.getContext && item.getContext('2d');
 
+		// Load platform resources on first chart creation, to make possible to change
+		// platform options after importing the library (e.g. `disableCSSInjection`).
+		this._ensureLoaded();
```

Comment:
As noted in PR description, injection is moved from `initialize` to `acquireContext`.
But what is the reason?

Reply:
Currently, the injection happens directly after the library is loaded, resulting to CSP errors if `unsafe_inline` is not allowed. To make it possible for the user to disable this injection, it would have to be done prior to loading the library (current situation), what would require a browser-global variable.

To use a Chart.js specific option, the injection has to be delayed after the library is loaded.

Maybe the comments needs to be more clear on this?

Reply:
`initialize()` is called when importing the library, which allow the platform to perform any initialization tasks *before the user reaches our API* (e.g. overrides, etc.). However, to give the user a chance to change the `disableCSSInjection` variable, we need to delay CSS injection until we actually need the CSS.

We could inject that CSS on the first call to `addResizeListener` but since the stylesheet could be used for something else than resizing, I feel `acquireContext` is a good spot to perform that loading.

Reply:
I'm one of those that have a preprocessor integrated in brain to remove comments from code 🙈 
The comment is clear enough, I just didn't read it.